getting_started/topics/first-boot/distribution-files-community.adoc

```diff
@@ -1,11 +1,28 @@
 
 === Installing Distribution Files
 
-The {project_name} Server is contained in one https://www.keycloak.org/downloads.html[distribution file]:
+Download the {project_name} Server:
 
-* 'keycloak-{project_version}.[zip|tar.gz]'
+* *keycloak-{project_version}.[zip|tar.gz]*
 
-The 'keycloak-{project_version}.[zip|tar.gz]' file is the server only distribution. It contains nothing other than the scripts and binaries
-to run the {project_name} server.
+NOTE: This file can be downloaded from https://www.keycloak.org/downloads.html[Keycloak downloads].
 
-To unpack these files run the `unzip` or `gunzip` and `tar` utilities.
+The *keycloak-{project_version}.[zip|tar.gz]* file is the server-only distribution. It contains only the scripts and binaries to run the {project_name} server.
+
+Place the file in a directory you choose and use either the `unzip` or `tar` utility to extract it.
+
+.Linux/Unix
+[source]
```

Comment:
Should be:
[source,bash,subs=+attributes]

Using bash highlights the code properly and subs=+attributes allows variables within source blocks

Reply:
Will push a new commit momentarily.